apprentice_teacher.rb

```diff
@@ -1,21 +1,10 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'educator'
+require_relative 'high_five'
+class ApprenticeTeacher < Educator
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super
     @target_raise = 800
```

Comment:
since this attribute shows up in two different staff members, how can you assign this attribute a value at the parent class but have separate values for each subclass?